english/texas/texas.md

```diff
@@ -0,0 +1,4 @@
+The man informs you that you are in Texas!!
+No wonder everything is so big.
+Time to follow your [destiny](../destiny.destiny.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.